showcase/src/styles/styles.pcss

```diff
@@ -3,7 +3,5 @@ IMPORTANT: Stark styles are provided as PostCSS styles so they should be importe
 */
 
 /* Stark variables */
-@import "../../node_modules/@nationalbankbelgium/stark-ui/assets/styles/_variables.pcss";
```

Comment:
Same as above, without this I guess the app-logo component won't be displayed correctly 😕